cmd/qgb/orchestrator/cmd.go

```diff
@@ -134,10 +177,41 @@ func Command() *cobra.Command {
 			// Listen for and trap any OS signal to gracefully shutdown and exit
 			go helpers.TrapSignal(logger, cancel)
 
+			// starting the orchestrator
 			orch.Start(ctx)
 
 			return nil
 		},
 	}
 	return addOrchestratorFlags(command)
 }
+
+// Init initializes the orchestrator store and creates necessary files.
+func Init() *cobra.Command {
```

Comment:
Now that we have the orchestrator starting in 2 steps: init then start. Then, we should update the dockerfile accordingly.

Reply:
https://github.com/celestiaorg/orchestrator-relayer/issues/253